packages/block-editor/tsconfig.json

```
@@ -0,0 +1,13 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types"
+	},
+	// NOTE: This package is being progressively typed. You are strongly
+	// encouraged to expand this array with files which can be type-checked. At
+	// some point in the future, this can be simplified to `src/**/*`.
+	"include": [
```

For _files_, we might just use `files` rather than `include`:
```suggestion
	"files": [
```

I understand `include` is really for globs.

---

My thinking was that if we consider this to be progressively opting-in, we could eventually expand this to include glob patterns of specific parts of the package, e.g. `src/utils/*`. I guess it wouldn't really be too much an issue to individually list the files, especially since they'd probably be added one at a time (the glob would be introduced at a point where a whole folder is "complete").

---

Addressed in 7f4a5a8db9.